src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/acl_editor.cljs

```diff
@@ -107,11 +141,14 @@
                   #(reduce
                     (fn [state [k v]]
                       (update state
-                              (if (= v "PROJECT_OWNER")
+                              (if (= (v "accessLevel") "PROJECT_OWNER")
                                 :project-owner-acl-vec
                                 :non-project-owner-acl-vec)
-                              conj {:email k :accessLevel v :read-only? true}))
+                              conj {:email k
+                                    :accessLevel (v "accessLevel")
+                                    :pending? (v "pending")
+                                    :read-only? true}))
                     (assoc % :project-owner-acl-vec []
                              :non-project-owner-acl-vec [])
-                    (get-parsed-response false)))
+                    ((get-parsed-response false) "acl")))
```

Comment:
I would probably prefer to keywordize the response, but it doesn't gain much here.